src/net/sourceforge/kolmafia/persistence/DebugDatabase.java

```diff
@@ -2608,8 +2608,9 @@ private static void checkPower(final int itemId, final boolean force) {
     }
   }
 
-  // Helper method to force normalize Concoction comparisons to [-1, 0, 1] before testing
-  private static int sgn(int value) {
+  // Helper method to force normalize Concoction comparisons to [-1, 0, 1] before testing contract.
+  // Visible for unit testing
```

Comment:
You could use `Integer.signum` instead? And also replace this sgn method with it.